l1-contracts/test/foundry/unit/concrete/Bridgehub/experimental_bridge.t.sol

```diff
@@ -320,6 +377,182 @@ contract ExperimentalBridgeTest is Test {
+            _stateTransitionManager: address(mockSTM),
+            _baseToken: address(testToken),
+            _salt: uint256(123),
+            _admin: admin,
+            _initData: bytes("")
+        });
+    }
+
+    function test_RevertWhen_STMNotRegisteredOnCreate(uint256 chainId) public {
+        address deployerAddress = makeAddr("DEPLOYER_ADDRESS");
+        admin = makeAddr("NEW_CHAIN_ADMIN");
+
+        vm.prank(bridgeOwner);
+        bridgeHub.setPendingAdmin(deployerAddress);
+        vm.prank(deployerAddress);
+        bridgeHub.acceptAdmin();
+
+        chainId = bound(chainId, 1, type(uint48).max);
+        vm.expectRevert("Bridgehub: state transition not registered");
+        vm.prank(deployerAddress);
+        bridgeHub.createNewChain({
+            _chainId: chainId,
+            _stateTransitionManager: address(mockSTM),
+            _baseToken: address(testToken),
+            _salt: uint256(123),
+            _admin: admin,
+            _initData: bytes("")
+        });
+    }
+
+    function test_RevertWhen_wrongChainIdOnCreate(uint256 chainId) public {
+        address deployerAddress = makeAddr("DEPLOYER_ADDRESS");
+        admin = makeAddr("NEW_CHAIN_ADMIN");
+
+        vm.prank(bridgeOwner);
+        bridgeHub.setPendingAdmin(deployerAddress);
+        vm.prank(deployerAddress);
+        bridgeHub.acceptAdmin();
+
+        chainId = bound(chainId, type(uint48).max + uint256(1), type(uint256).max);
+        vm.expectRevert("Bridgehub: chainId too large");
+        vm.prank(deployerAddress);
+        bridgeHub.createNewChain({
+            _chainId: chainId,
+            _stateTransitionManager: address(mockSTM),
+            _baseToken: address(testToken),
+            _salt: uint256(123),
+            _admin: admin,
+            _initData: bytes("")
+        });
+
+        chainId = 0;
+        vm.expectRevert("Bridgehub: chainId cannot be 0");
+        vm.prank(deployerAddress);
+        bridgeHub.createNewChain({
+            _chainId: chainId,
+            _stateTransitionManager: address(mockSTM),
+            _baseToken: address(testToken),
+            _salt: uint256(123),
+            _admin: admin,
+            _initData: bytes("")
+        });
+    }
+
+    function test_RevertWhen_tokenNotRegistered() public {
+        address deployerAddress = makeAddr("DEPLOYER_ADDRESS");
+        admin = makeAddr("NEW_CHAIN_ADMIN");
+
+        vm.prank(bridgeOwner);
+        bridgeHub.setPendingAdmin(deployerAddress);
+        vm.prank(deployerAddress);
+        bridgeHub.acceptAdmin();
+
+        vm.startPrank(bridgeOwner);
+        bridgeHub.addStateTransitionManager(address(mockSTM));
+        vm.stopPrank();
+
+        vm.expectRevert("Bridgehub: token not registered");
+        vm.prank(deployerAddress);
+        bridgeHub.createNewChain({
+            _chainId: 1,
+            _stateTransitionManager: address(mockSTM),
+            _baseToken: address(testToken),
+            _salt: uint256(123),
```

Comment:
Please fuzz the tests as much as you can. In this case, try and provide the _chainId parameter as a fuzzed number bounded to be between 1 and type(uint48).max and similarly for the `_salt` parameter.